### PR TITLE
Add source-maps and cleanup repo/package size.

### PR DESCRIPTION
**Pull request by jgerigmeyer**
- Add source-maps to JS/CSS assets
- Remove built files from the repo
- Remove unnecessary files from NPM package

### CODE REVIEW

**Review by jerivas — APPROVED** [review]
So much cleaner! 🚀

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,7 +15,10 @@
   - 🚀 NEW: `utilities.each-key()` will pass each _key_ of a map
     through a given function
 - 🚀 NEW: Allow `utilities.add()` to merge data, instead of override
+- 🚀 NEW: Include source-maps with JS & CSS assets
```
Comment (mirisuzanne) [inline CHANGELOG.md:18]: Is this user-facing? What does it mean in practice?

**Review by mirisuzanne — APPROVED** [review]
looks good to me

File: CHANGELOG.md (hunk 1 above)
Comment (jgerigmeyer) [reply CHANGELOG.md:18]: These are the JS and CSS files that are served as part of Herman itself (not user-provided). I guess it's maybe more `INTERNAL` than `NEW` -- but it is a change in production for us to now include source-maps for the Herman assets themselves.

Comment (mirisuzanne) [reply CHANGELOG.md:18]: Got it. Sounds good.

**Review by mirisuzanne — APPROVED** [review]

**Pull request merged**